public/bz_support.js

```diff
@@ -290,3 +290,116 @@ function bzInitializeInstantSurvey() {
 	http.send();
 
 }
+
+function BZ_ModalDialog(titleString, bodyElement, onOK) {
+  var holder = document.createElement("div");
+  holder.setAttribute("id", "bz-fullscreen_modal");
+  var inner = document.createElement("div");
+  holder.appendChild(inner);
+
+  var title = document.createElement("h1");
+  title.textContent = titleString;
+  inner.appendChild(title);
+
+  var bodyHolder = document.createElement("div");
+  bodyHolder.className = "bz-fullscreen_modal-body_holder";
+  inner.appendChild(bodyHolder);
+
+  var buttonsHolder = document.createElement("div");
+  buttonsHolder.className = "bz-fullscreen_modal-buttons_holder";
+  inner.appendChild(buttonsHolder);
+
+  var cancelButton = document.createElement("button");
+  cancelButton.setAttribute("type", "button");
+  cancelButton.className = "bz-fullscreen_modal-cancel_button";
+  cancelButton.textContent = "Cancel";
+  cancelButton.onclick = function() {
+    document.body.removeChild(holder);
+  };
+  buttonsHolder.appendChild(cancelButton);
+
+  var okButton = document.createElement("button");
+  okButton.setAttribute("type", "button");
+  okButton.className = "bz-fullscreen_modal-ok_button";
+  okButton.textContent = "OK";
+  okButton.onclick = function() {
+    onOK();
+    document.body.removeChild(holder);
+  };
+  buttonsHolder.appendChild(okButton);
+
+  document.body.appendChild(holder);
+
+  bodyHolder.appendChild(bodyElement);
+}
+
+var BZ_MasterBankCourseId = 28;
```

Comment:
Can we hardcode this to 1?  Looks like there isn't a course with ID 1 yet and I think that makes more sense.  Unless ID 1 is somehow already a magic number.